tests/attacks/evasion/test_imperceptible_asr.py

```diff
@@ -0,0 +1,313 @@
+# MIT License
+#
+# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2020
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+
+import numpy as np
+import pytest
+from numpy.testing import assert_array_equal
+
+from art.attacks.attack import EvasionAttack
+from art.attacks.evasion.imperceptible_asr.imperceptible_asr import ImperceptibleAsr, PsychoacousticMasker
+from tests.utils import ARTTestException
+
+logger = logging.getLogger(__name__)
+
+
+class TestImperceptibleAsr:
+    """
+    Test the ImperceptibleAsr attack.
+    """
+
+    @pytest.mark.skipMlFramework("pytorch", "tensorflow1", "mxnet", "kerastf", "non_dl_frameworks")
```

Comment:
This marker applies to all tests. Is it possible mark the test class with it?

Reply:
Yes, you are right. That is possible. Previously, I also asked @killianlevacher about this but he recommended that we always put markers at the method/function-level as this advised in the guidelines.